src/app/api/comment/comment.service.ts

```diff
@@ -41,6 +43,23 @@ export class CommentService extends BaseService {
       }),
     )
 
+    if (comment) {
+      const task = await this.db.task.findFirst({
+        where: {
+          id: data.taskId,
+        },
+      })
+      if (task) {
+        const notificationService = new NotificationService(this.user)
+        if (task?.assigneeId) {
```

Comment:
do we need the optional chaining here @aatbip ?

Reply:
we shouldn't need it here. Also isn't this an error scenario? if task is not found than it doesn't even make sense to create the comment right?

Reply:
Fixed in 3a9dcb86db254fe7b133359ecb68d8b0b1d765be commit